bootstrapping_decision_making.md

```diff
@@ -0,0 +1,33 @@
+- We need to take existing Steering Council members into account. There may be Steering Council members that would like to be on the formal decision making body for a Subproject, even though they have not been active on the Suproject for a period of time.
+
+## Proposed framework
+
+We would like to propose the following framework for Subprojects to establish and bootstrap their formal decision making bodies. This framework is not being presented as a set of strict rules, but more as a set of flexible principles that Subprojects can use for this task. The Governance Working Group will be available to meet with all Subprojects to openly discuss how the Subproject can best use and adapt these ideas to their case.
+
+The overall principle of this framework is that of gradual accumulation of the decision making body, rather than bulk election. The process we propose can be briefly outlined as follows:
+
+
+- Each Subproject typically has a core of individuals who have a long history with the Subproject and have made significant contributions to the Subproject (in many cases, they founded the Subproject). There is universal agreement that these people should be on the formal decision making body for the Subproject. An example would be Min for JupyterHub. This initial group can be as small as a single person and keeping it as small as possible ensures this step will be simple. Let’s call the size of this initial group `N0`.
+- Once the initial group or person is established, that group will use the new Decision Making Guidelines to elect the next single member they would like to add to the formal decision making body. At this point, the decision making body would have size `N0+1`.
+- At this point, the `N0+1` people would use the Decision Making Guidelines to elect the next single member to add to the decision make body.
+- This process continues in an inductive manner where at each point, the current `N` people on the decision making body elect the `N+1` person, or choose to stop electing new members.
+
+This framework has the following characteristics:
+
+- By gradually increasing the size of the decision making body one person at a time, the power implicit in this election process is gradually distributed over a larger group of people.
+- At each stage where the new decision making body is considering adding a new person, they can also choose to stop electing additional members. This allows the group to decide on the overall size in an organic, gradual manner.
+- The bootstrapping process should address issues of multi-stakeholder governance. Specifically, during bootstrapping, decision making bodies should rotate new members across different organizations to avoid overrepresentation of a single organization.
+- Decision making bodies should be inclusive and as large as necessary to give voice to contributors who are actively working on the subproject or working group.
```

Comment:
It would be great to have a step in the process above to check that being inclusive is happening as the subproject adds members. If not, an effort should be made to do so.

Failure to make inclusion part of the process will result in the status quo or possibly less inclusion.

Reply:
agree - telling someone they should be inclusive is not a guarantee at all that it will actually happen. It will take constant attention and effort, and occasionally uncomfortable conversations that likely involve power differentials. How can we ensure that, in those conversations, the person representing the marginalized perspective is empowered to feel confident in pushing for more diverse representation?

I’d also like to see the tangible efforts that the Jupyter community will make towards providing support or training that encourages more inclusive governance and participation. building more inclusive dynamics takes work and skills that many oss devs do not have. (If it is too early to know these specific things, perhaps this document can empower a subgroup with authority and resources to do this?)